### PR TITLE
Restructured readme and added changelogs

### PR DESCRIPTION
**Pull request by Brobin**
Added links to navigate the readme as well as the release notes at the bottom for a quick reference to changes.

### CODE REVIEW

**Comment by michael-lazar** [comment]
From what I've seen the standard convention is to include changes in a seperate document and link to them in the README. Here are some examples that I found going through the top Python github repos.

https://github.com/kennethreitz/requests  ``HISTORY.rst``
https://github.com/mitsuhiko/flask ``CHANGES``
https://github.com/jakubroztocil/httpie ``CHANGELOG.rst``
https://github.com/getsentry/sentry ``CHANGES``

Check out [httpie's changelog](https://github.com/jakubroztocil/httpie/blob/master/CHANGELOG.rst), I think theirs is especially well done. Actually I wouldn't mind straight up copying httpie's README format all together. I like having the picture/summary at the very top.

```
summary
picture
sheilds
toc
contents
...
authors
contribute
changelog
license
```

I also drew README inspiration from this project https://github.com/ryanss/vim-hackernews

**Comment by Brobin** [comment]
Actually, I really like the idea of separate files. How about this?

`README.rst` - General readme stuff, links to others
`HISTORY.rst` - Document changes, link to version tags
`CONTRIBUTORS.rst` - List of contributors

**Comment by michael-lazar** [comment]
Yea I like that!

**Comment by Brobin** [comment]
Do you want to leave usernames on the changelog?

**Comment by michael-lazar** [comment]
We don't have to as long as there's a seperate CONTRIBUTORS file

**Comment by Brobin** [comment]
That's what I was thinking

**Comment by Brobin** [comment]
Sorry for the superfluous number of commits. I get trigger-happy with commits late at night :P

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+=============
+RTV Changelog
+=============
+
+.. _1.2.1: http://github.com/michael-lazar/rtv/releases/tag/v1.2.1
+.. _1.2: http://github.com/michael-lazar/rtv/releases/tag/v1.2
+
+------
+1.2.1_
```
Comment (michael-lazar) [inline CHANGELOG.rst]: Can you add dates to the release numbers? Besides that, this looks damn good :+1:

Comment (michael-lazar) [inline CHANGELOG.rst]: Nevermind, did it myself

Comment (Brobin) [inline CHANGELOG.rst]: Great!

**Pull request merged**